src/txmempool.cpp

```diff
@@ -1150,3 +1227,169 @@ size_t CTxMemPool::DynamicMemoryUsage() const {
           cachedInnerUsage);
 }
 
+bool CCoinsViewMemPool::IsQualityValid(const CScCertificate& cert, CAmount certFee) const
```

Comment:
As mentioned in coins.cpp file, I would rename it like `CheckQuality(cert)` or `HaveQuality(cert)`

Reply:
Done